core/tests/protractor_utils/PreferencesPage.js

```diff
@@ -107,9 +107,15 @@ var PreferencesPage = function() {
   };
 
   this.setUserBio = async function(bio) {
+    waitFor.visibilityOf(
```

Comment:
Is this an expected change for this PR?

Reply:
Not really, my thought was that I could reduce an extra PR build in the queue. Would you like me to raise this as a separate fix?

Reply:
No, I'm fine with it! I just wanted to make sure that its expected!